source/Interface.cpp

```diff
@@ -25,6 +25,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Information.h"
 #include "text/layout.hpp"
 #include "LineShader.h"
+#include "MapPanel.h"
```

Comment:
I'm really not a fan of including "MapPanel.h" in "Interfaces.cpp", especially if it's to support an interface definition having a property that depends entirely on the contents of that class.
The "map color" values are entirely meaningless to someone looking at the interfaces file, or even the class declaration ffor that interface element.
I would suggest instead that each ring be given a unique name and support for setting the colour of bar elements via interface `Information` objects be added. That way, the ring can be named something like "sold here" and MapSalesPanel::DrawKey can set the colour for rings with that name.